### DISCUSSION TITLE
Couldnt use copilot

### DISCUSSION THREAD

**Discussion by oguzbyk** [community/community#43469] [Copilot Conversations]
### have problem about using copilot

Product Feedback

### 

Today I get a subscription to copilot. I install copilot to IntelliJ and login with GitHub account. When I open the copilot view it says "Refresh to get completions for the caret positions." When I press the Refresh button It says "Retrieving completions" I wait for more than ten minutes but nothing happens.
Also, there is a little icon right button on the screen that has "!"  and says "copilot is encountering temporary issues".

Btw in IntelliJ, I have a company GitLab account and all projects is connected to that. But my copilot settings belong to my GitHub account.
So can anyone help me please?

**Comment by CarlinJunHuang** [comment]
😭
